package.json

```diff
@@ -43,7 +43,8 @@
     "fiori-fundamentals": "^1.4.0",
     "react-router": "^4.3.1",
     "react-router-dom": "^4.3.1",
-    "react-syntax-highlighter": "^9.0.1"
+    "react-syntax-highlighter": "^9.0.1",
+    "shortid": "^2.2.14"
```

Comment:
shortid can generate invalid HTML id. Use a util function to build HTML valid shortid values

see: https://github.concur.com/coreui/nui-widgets/pull/432/files

Reply:
Fixed in c25e564